frontend/public/style/_layout.scss

```diff
@@ -3,7 +3,8 @@ body,
 #app,
 #content,
 #content-scrollable,
-.pf-c-page__main-section {
+.pf-c-page__main-section,
+.pf-c-drawer {
```

Comment:
If this might need to go along with the drawer, we should consider putting it in its own file.  Ideally it would exist with the component in your package, but I don't think we are setup to scan for scss like that.